docs/UserGuide.md

```diff
@@ -115,46 +115,46 @@ section of this user guide or to an external website.
+   * Alternatively, if you wish to use the `Terminal` or `Command Prompt`, navigate to the **home folder** and enter 
    `java -jar guestbook.jar` to start the app. 
 
    The GUI, similar to the image below, should pop up in a few seconds. Note that the GuestBook
    comes prepackaged with **sample data**.<br><br>
    ![Ui](images/Ui.png) <br><br>
 
-5. Type the command in the command box and press Enter to execute it. E.g., typing **`help`** and pressing Enter will open the help window.<br><br>
+5. Type your commands in the command box and press Enter to execute it. E.g., typing **`help`** and pressing Enter will open the help window.<br><br>
 
 6. To kickstart your journey, you can add a guest, _John Doe_, to GuestBook using the **`add`** command.
-   * **`add`** `n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named _John Doe_ to GuestBook. 
+   * `add n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named _John Doe_ to GuestBook. 
 <br><br>
     
 7. To find _John Doe_, you can use the **`find`** command, and search through **any of _John Doe's_ fields**,
    such as his `name` or `room`. The command below can help you find guests that match the keywords you entered. For example,
-   * **`find`** `John Doe` : Finds all the guests who have at least one field containing _John_ or _Doe_ (case-insensitive).
+   * `find John Doe` : Finds all the guests who have at least one field containing _John_ or _Doe_ (case-insensitive).
      <br><br>
 
 8. To make changes to _John Doe_, you can use the **`edit`** command. This changes the guest's fields based on the new values you provide. The other values will remain unchanged. <br>
-For example, if _John Doe_ is the second guest in the list, and you want to change his room number to _05-55_, you can do this.
-    * **`edit`** `2` `rm/05-55`: Changes the `Room` of _John Doe_ to _05-55_ in GuestBook.
+For example, if _John Doe_ is the second guest in the list, and you want to change his room number to _05-55_, you can execute the following command.
+    * `edit 2 rm/05-55`: Changes the `Room` of _John Doe_ to _05-55_ in GuestBook.
       <br><br>
    
 9. After executing a **`find`** command, you might realise that some of your guests are no longer visible, as they are not in the current list.<br>
 To see a list of all your guests, you can use the **`list`** command.
-      * **`list`**: List all the guests in GuestBook.
+      * `list`: List all the guests in GuestBook.
         <br><br>
 
 10. In the event that a guest orders room service, you can add charges to the guest's bill using the **`bill`** command.<br>
 For example, you can charge the third guest _30.35_ by running the command below.
-    * **`bill`** `3` `b/+30.35`: Adds _30.35_ to the `bill` of the third guest.
+    * `bill 3 b/+30.35`: Adds _30.35_ to the `bill` of the third guest.
```

Comment:
Should we add "shown in the current list" (similar to the delete below)?

Reply:
Agreed!